<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditNode</class>
 <widget class="QDialog" name="EditNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>530</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>530</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="StartupEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>561</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="ShutdownEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>541</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StartupLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Startup</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShutdownLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
  </widget>
  <widget class="QComboBox" name="NetworkSelect0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="NetworkSelect1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="NetworkSelect2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
